web/api/web_api_v1.c

```diff
@@ -1050,11 +1050,15 @@ inline int web_client_api_request_v1_info_fill_buffer(RRDHOST *host, BUFFER *wb)
     buffer_strcat(wb, ",\n");
 
 #ifdef  ENABLE_COMPRESSION
-    buffer_strcat(wb, "\t\"stream-compression\": ");
-    buffer_strcat(wb, (host->sender->rrdpush_compression ? "\"enabled\"" : "\"disabled\""));
-    buffer_strcat(wb, ",\n");
+    if(host->sender){
+        buffer_strcat(wb, "\t\"stream-compression\": ");
+        buffer_strcat(wb, (host->sender->rrdpush_compression ? "true" : "false"));
+        buffer_strcat(wb, ",\n");
+    }else{
+        buffer_strcat(wb, "\t\"stream-compression\": null,\n");
```

Comment:
should it be `null` here or `false`? Do we want `null` to mean -> not supported in this agent?

Reply:
Either this flag says:
- "true" compression is used now
- "false" compression is not used now
- "null" this agent has no compression

or what you suggest quote:
>  It means that it can be enabled or disabled but we don't have any info about it.

In such case it should reflect if it is enabled in configuration file. In such case `we don't have any info about it` makes no sense because we can read config key value regardless of `host->sender` existence.

Reply:
BTW, I suggested a more comprehensive solution in the [previous ](https://github.com/netdata/netdata/pull/12565#discussion_r838862644)comment